### DISCUSSION TITLE
Is there a way to find out the length of the Semaphore queue?

### DISCUSSION THREAD

**Discussion by guskovd** [tokio-rs/tokio#4283] [General]
Is there a way to find out the length of the Semaphore queue?

I found something similar in `batch_semaphore`: https://github.com/tokio-rs/tokio/blob/03969cdae7674681d1b10926e6a56fbb8908dbb8/tokio/src/sync/batch_semaphore.rs#L35

But as I understand it, this interface is not accessible from the outside.

**Comment by Darksonn** [comment]
No, there's is currently no such method.
